src/tests.builds

```diff
@@ -13,7 +13,10 @@
   <Import Project="$(ToolsDir)VersionTools.targets" Condition="Exists('$(ToolsDir)VersionTools.targets')" />
 
   <ItemGroup>
-    <Project Include="$(MSBuildThisFileDirectory)*\tests\**\*.Tests.csproj" Exclude="@(TestProjectExclusions)">
+    <Project Condition="'$(Performance)'=='true'" Include="$(MSBuildThisFileDirectory)System.Collections*\tests\**\*Performance.Tests.csproj" Exclude="@(TestProjectExclusions)">
```

Comment:
We have an entire configuration and test system that allows tests to be marked as supporting things we should follow a similar pattern for performance tests. I would be fine if you use some file naming convention like ends with "Performance.Tests" or something but I don't want test.builds to turn into a large list of projects.